.circleci/config.yml

```diff
@@ -57,6 +70,21 @@ jobs:
           paths:
             - docs/docs/_build/html/
 workflows:
+  # default trigger on push, don't double up with actions
   build-docs:
+    when:
+      not: << pipeline.parameters.GHA_Meta >>
```

Comment:
This change is needed to prevent double-triggers. We want the normal workflow to trigger on push/commit/PR status only, not when the comment/trigger is used.